src/Ae.js

```diff
@@ -84,30 +84,36 @@ export default class Ae {
     rawTxHex: string,
     networkId: string = "ae_mainnet"
   ): Promise<string> {
+    let isNewAppVersion = true;
+    const appConfig = await this.transport.send(CLA, GET_APP_CONFIGURATION, 0x00, 0x00);
+    if (appConfig[1] === 0 && appConfig[2] < 4) {
+      isNewAppVersion = false;
+    }
     let offset = 0;
+    const indent = 4 + 1 + (isNewAppVersion ? 4 : 0);
```

Comment:
`indent` sounds similar to `offset` that is used near, maybe name it like `headerLength` instead?